src/core/services/gesture/gesture.js

```diff
@@ -302,7 +302,7 @@ angular.module('material.core')
     onCancel: angular.noop,
     options: {},
 
-    dispatchEvent: dispatchEvent,
+    dispatchEvent: typeof jQuery !== 'undefined' && angular.element === jQuery ? jQueryDispatchEvent : nativeDispatchEvent,
```

Comment:
It has been merged, btw. The problem is that `typeof jQuery !== 'undefined' && angular.element === jQuery` might evaluate to false (thus using `nativeDispatchEvent` although jQuery **is** used (leading to the wrong behaviour again)